SPECS/avahi/avahi.spec

```
@@ -420,6 +425,21 @@ exit 0
 %endif
 
 %changelog
+* Mon Dec 02 2024 Kanishk Bansal <kanbansal@microsoft.com> -0.8-4
+- Fix CVE-2023-38471 wih an upstream patch
+
+* Mon Dec 02 2024 Kanishk Bansal <kanbansal@microsoft.com> -0.8-4
+- Fix CVE-2023-38470 with an upstream patch
+
+* Mon Dec 02 2024 Kanishk Bansal <kanbansal@microsoft.com> -0.8-4
+- Fix CVE-2023-38473 with an upstrem patch
+
+* Mon Dec 02 2024 Kanishk Bansal <kanbansal@microsoft.com> -0.8-4
+- Fix CVE-2023-38472 with an upstream patch
+
+* Wed Nov 27 2024 Kanishk Bansal <kanbansal@microsoft.com> - 0.8-4
+- Fix CVE-2023-38469 with an upstream patch
+
```

Usually we list multiple changes as single changelog entry within same release using the - as list entries like this
```suggestion
* Wed Nov 27 2024 Kanishk Bansal <kanbansal@microsoft.com> - 0.8-4
- Fix CVE-2023-38469 with an upstream patch
- Fix CVE-2023-38470 with an upstream patch
- Fix CVE-2023-38471 with an upstream patch
- Fix CVE-2023-38472 with an upstream patch
- Fix CVE-2023-38473 with an upstream patch
```

---

done